yew-ui/src/model/media_devices/media_device_access.rs

```diff
@@ -0,0 +1,62 @@
+use gloo_utils::window;
+use std::cell::Cell;
+use std::sync::Arc;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::JsFuture;
+use web_sys::MediaStreamConstraints;
+use yew::prelude::Callback;
+
+pub struct MediaDeviceAccess {
+    granted: Arc<Cell<bool>>,
```

Comment:
@ronen what's the advantage of `Arc<Cell>` vs AtomicBool?

Reply:
None that I can think of :)   Switched to `AtomicBool` (2ac3fa9)